app/frontend/src/components/EnvironmentBanner.tsx

```diff
@@ -25,17 +25,13 @@ export function EnvironmentBanner() {
   const isBelowSm = useMediaQuery(theme.breakpoints.down("sm"));
   const [isHidden, setIsHidden] = useState(false);
```

Comment:
const [isHidden, setIsHidden] = useState(process.env.REACT_APP_COUCHERS_ENV !== "preview");

Reply:
Nice suggestion, actually this gives a good opportunity to simplify the render logic too and I flipped it so it reads more naturally (true renders, false renders nothing)